test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/AuthSupport.scala

```diff
@@ -35,6 +38,17 @@ trait AuthSupport {
 
   val mockAuthConnector: AuthConnector             = mock[AuthConnector]
   val mockSubscriptionService: SubscriptionService = mock[SubscriptionService]
+  def mockGetSubscriptionDetails(cgtReference: CgtReference, expectedSubscribedDetails: SubscribedDetails): Unit =
+    (mockSubscriptionService
+      .getSubscribedDetails(_: CgtReference)(_: HeaderCarrier))
+      .expects(cgtReference, *)
+      .returning(
+        EitherT[Future, Error, SubscribedDetails](
+          Future.successful(
+            Right(expectedSubscribedDetails)
```

Comment:
how are you testing the error cases if you can't simulate an error here?

Reply:
Ok, parameterised, and added those missing tests too.